sagemaker/01_getting_started_pytorch/scripts/train.py

```diff
@@ -59,7 +60,7 @@ def compute_metrics(pred):
 
     # define training args
     training_args = TrainingArguments(
-        output_dir=args.model_dir,
+        output_dir=args.output_dir,
```

Comment:
[Trainer by default saves all checkpoints](https://huggingface.co/docs/transformers/main_classes/trainer#checkpoints) at `output_dir`. Previously, both the model and the checkpoints were being saved at `args.model_dir` i.e. `SM_MODEL_DIR`. This fix saves checkpoints at `/opt/ml/checkpoints`.